content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Minimal Istio Installation
```

Comment:
I like the structure.  We can then put these in something like advanced_install/minimal-install along with the other install recipes we recommend.

Nice start 👍

Reply:
Thanks!